src/index.js

```diff
@@ -68,118 +63,143 @@ query ($id: ID!) {
     }
   }
 }
-`;
+`
+/**
+ * Sends POST request to telegram bot
+ * @param {String} data - telegram message
+ */
+async function notify (data) {
+  return axios({
+    method: 'POST',
+    url: NOTIFIER_URL,
+    data: 'message=' + encodeURIComponent(data)
+  })
+}
 
-function extractPullRequests(cards) {
-  const prs = new Set();
-  for (let card of cards) {
-    card = card.content;
-    if (!card) continue;
-    if (card.__typename === "Issue") {
-      for (const event of card.timelineItems.nodes) {
-        if (event.source) {
-          switch (event.__typename) {
-            case "ConnectedEvent":
-              prs.add(event.subject.url);
-              break;
-            case "CrossReferencedEvent":
-              prs.add(event.source.url);
-              break;
-          }
-        }
+/**
+ * Parse the response of spring backlog query.
+ * @param {Array} members - array of object contains members list
+ * @param {Array} response - response of query as array of object
+ * @returns {Array} - array of object which contains members with task
+ */
+function parseQuery (members, response) {
+  const data = response.map((items) => {
+    if (items.state === 'NOTE_ONLY') {
+      return (items.note)
+    } else if (items.state === 'CONTENT_ONLY') {
+      if (items.content.__typename === 'PullRequest') { return ('@' + items.content.author.login + ' ' + items.content.url) }
+      if (items.content.__typename === 'Issue') {
+        const people = items.content.assignees.nodes.map((item) => {
+          return '@' + item.login + ' '
```

Comment:
more template strings or string literals, how are they called in JS

Reply:
string Literals